resources/views/dashboard/widgets/forms/_recent.blade.php

```diff
@@ -80,6 +80,14 @@
             ], null, ['class' => '']) !!}
                 </x-forms.field>
                 @includeWhen(!empty($dashboards), 'dashboard.widgets.forms._dashboard')
+
+                <x-forms.field field="config[created_by]" :label="__('dashboard.widgets.fields.created_by')">
```

Comment:
We already have a filters field, we need to use that instead, by just adding the parameters there